sklearn/ensemble/voting.py

```diff
@@ -304,42 +364,125 @@ class labels predicted by each classifier.
+    estimators : list of (string, estimator) tuples
+        Invoking the ``fit`` method on the ``VotingRegressor`` will fit
+        clones of those original estimators that will be stored in the class
+        attribute ``self.estimators_``. An estimator can be set to `None`
+        using ``set_params``.
+
+    weights : array-like, shape = [n_regressors], optional (default=`None`)
+        Sequence of weights (`float` or `int`) to weight the occurrences of
+        predicted values before averaging. Uses uniform weights if `None`.
+
+    n_jobs : int or None, optional (default=None)
+        The number of jobs to run in parallel for ``fit``.
+        ``None`` means 1 unless in a :obj:`joblib.parallel_backend` context.
+        ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
+        for more details.
+
+    Attributes
+    ----------
+    estimators_ : list of regressors
+        The collection of fitted sub-estimators as defined in ``estimators``
+        that are not `None`.
+
+    named_estimators_ : Bunch object, a dictionary with attribute access
+        Attribute to access any fitted sub-estimators by name.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.linear_model import LinearRegression
+    >>> from sklearn.ensemble import RandomForestRegressor
+    >>> from sklearn.ensemble import VotingRegressor
+    >>> r1 = LinearRegression()
+    >>> r2 = RandomForestRegressor(n_estimators=10, random_state=1)
+    >>> X = np.array([[1, 1], [2, 4], [3, 9], [4, 16], [5, 25], [6, 36]])
+    >>> y = np.array([2, 6, 12, 20, 30, 42])
+    >>> er = VotingRegressor([('lr', r1), ('rf', r2)])
+    >>> print(er.fit(X, y).predict(X))
+    [ 3.3  5.7 11.8 19.7 28.  40.3]
+
+    See also
+    --------
+    VotingClassifier: Soft Voting/Majority Rule classifier.
+    """
+
+    def __init__(self, estimators, weights=None, n_jobs=None):
+        self.estimators = estimators
+        self.weights = weights
+        self.n_jobs = n_jobs
+
+    def fit(self, X, y, sample_weight=None):
+        """ Fit the estimators.
 
         Parameters
         ----------
-        **params : keyword arguments
-            Specific parameters using e.g. set_params(parameter_name=new_value)
-            In addition, to setting the parameters of the ``VotingClassifier``,
-            the individual classifiers of the ``VotingClassifier`` can also be
-            set or replaced by setting them to None.
+        X : {array-like, sparse matrix}, shape = [n_samples, n_features]
+            Training vectors, where n_samples is the number of samples and
+            n_features is the number of features.
 
-        Examples
-        --------
-        # In this example, the RandomForestClassifier is removed
-        clf1 = LogisticRegression()
-        clf2 = RandomForestClassifier()
-        eclf = VotingClassifier(estimators=[('lr', clf1), ('rf', clf2)]
-        eclf.set_params(rf=None)
+        y : array-like, shape = [n_samples]
```

Comment:
```suggestion
        y : array-like, shape (n_samples,)
```

Reply:
this is a new guide for documenting?
I looked in other files. there used `= []` style

Reply:
Yes some other files still use `[]` but we're encouraging the use of parenthesis for new code (see e.g. the contirbuting guide)